gcloud_dataproc/load_dataset.py

```diff
@@ -290,50 +369,60 @@ def main():
             run("gsutil cp %(subset_samples_file_path)s %(subset_samples_file_gcloud_path)s" % locals())
             load_dataset_to_es_args.extend(["--subset-samples", subset_samples_file_gcloud_path])
 
-    # make sure kubectl is installed
-    run("kubectl version --client")
-
-    # make sure cluster exists
-    _create_dataproc_cluster(
-        args.cluster_name,
-        args.genome_version,
-        num_workers=args.num_workers,
-        num_preemptible_workers=args.num_preemptible_workers)
-
-
     # run pipeline with or without using a temp elasticsearch cluster for loading
-    if args.use_temp_loading_nodes:
+    if args.use_temp_loading_nodes and (args.stop_after_step == None or args.stop_after_step > 1):
+        # make sure kubectl is installed
+        run("kubectl version --client")
+
 
         # run vep and compute derived annotations before create temp elasticsearch loading nodes
         if args.start_with_step <= 1:
+            # make sure cluster exists
+            _create_dataproc_cluster(
+                args.cluster_name,
+                args.genome_version,
+                num_workers=args.num_workers,
+                num_preemptible_workers=args.num_preemptible_workers)
             submit_load_dataset_to_es_job(
                 args.cluster_name,
                 start_with_step=args.start_with_step,
                 stop_after_step=1,
                 other_load_dataset_to_es_args=load_dataset_to_es_args)
-
-        _enable_cluster_routing_rebalance(False, args.cluster_name, args.host, args.port)
-
         # create temp es nodes
         settings = {
             "DEPLOY_TO": args.k8s_cluster_name,
             "CLUSTER_NAME": args.k8s_cluster_name,
             "ES_CLUSTER_NAME": args.k8s_cluster_name,
             "NAMESPACE": args.k8s_cluster_name,  # kubernetes namespace
             "IMAGE_PULL_POLICY": "Always",
+            #"GCLOUD_ZONE": "central",
```

Comment:
why is this commented out?

Reply:
it's typically already configured in someone's local gcloud config.

Reply:
but then why not just remove it?